### PR TITLE
Add a test to make sure that DRF docs work

### PR DESCRIPTION
**Pull request by akalipetis**
When the DRF docs try to inspect the UserViewSet, get_serializer_class
crashes because there's no request present. In such case, we should
fallback to the default serializer and everything should work as expected.

Signed-off-by: Antonis Kalipetis <akalipetis@gmail.com>

### CODE REVIEW

**Comment by akalipetis** [comment]
Not sure which is the best way to install dependencies, while docs say to use `pipenv`, Travis is using `requirements.txt`.

I have added `coreapi` to both `Pipfile` and `requirements.txt` and also locked `Pipfile.lock` again. Let me know if you'd like any changes in the code or setup 😀

File: djoser/views.py [hunk 1]
```diff
@@ -285,12 +285,14 @@ def get_permissions(self):
     def get_serializer_class(self):
         if self.action == 'create':
             return settings.SERIALIZERS.user_create
-        elif self.action == 'remove' or (
-                self.action == 'me' and self.request.method == 'DELETE'):
+        if self.action == 'remove' or (
```
Comment (dekoza) [inline djoser/views.py]: what's wrong with `elif`?

**Comment by dekoza** [comment]
Thanks for your time :)

**Comment by akalipetis** [comment]
Thanks a lot for a great library!

**Pull request merged**